actors/evm/tests/calc.rs

```diff
@@ -0,0 +1,159 @@
+mod asm;
+
+use evm::interpreter::U256;
+use fil_actor_evm as evm;
+use fil_actors_runtime::test_utils::*;
+use fvm_ipld_encoding::RawBytes;
+
+#[allow(dead_code)]
+pub fn magic_calc_contract() -> Vec<u8> {
+    let init = r#"
+push1 0x42  # magic value
+push1 0x00  # key of magic value
+sstore
+"#;
+    let body = r#"
```

Comment:
https://github.com/vorot93/evmodin/blob/master/tests/execute.rs here are more elegant approaches I think. You will not be testing entire programs but targeted bytecode snippets in the unit tests.

Reply:
we actually want to test entire contracts.